src/nidcpower/metadata/functions_addon.py

```diff
@@ -149,7 +149,7 @@
     },
     # Public function that wraps driver function but returns datetime object instead of individual items
     'GetLastExtCalLastDateAndTime': {
-        'codegen_method': 'public',
+        'codegen_method': 'api',
```

Comment:
I think `python-only` would be more self-explanatory. `api` doesn't make it clear if it's the Python bindings or the C API.

Also: Don't forget to update the wiki once this is merged!

Reply:
Changed to `'python-only'`